src/app.js

```diff
@@ -111,54 +59,17 @@ app.on('ready', () => {
       { label: `Best Streak: ${bestStreak}`, enabled: false },
       { label: `Contributions: ${contributionCount}`, enabled: false },
       { type: 'separator' },
-      { label: 'Reload', accelerator: 'Cmd+R', click: requestContributionData },
+      { label: 'Reload', accelerator: 'CmdOrCtrl+R', click: requestContributionData },
       {
         label: 'Open GitHub Profile...',
-        accelerator: 'Cmd+O',
+        accelerator: 'CmdOrCtrl+O',
         click: () => shell.openExternal(githubProfileUrl)
       },
       { type: 'separator' },
-      {
-        label: 'Set GitHub Username...',
-        accelerator: 'Cmd+S',
-        click: createUsernameWindow
-      },
       {
         label: 'Preferences',
```

Comment:
Most applications tend to suffix this with a "..." to let the user know that they will be taken elsewhere.